src/Lib/BSP/IO/BspLumpHeader.cs

```diff
@@ -0,0 +1,19 @@
+namespace Lumper.Lib.BSP.IO
+{
+    public class BspLumpHeader
+    {
+        public BspLumpHeader(LumpHeader header, int version)
+        {
+            Offset = (int)header.Offset;
+            Length = (int)header.Length;
+            Version = version;
+            FourCc = header.Compressed
+                        ? (int)header.UncompressedLength
+                        : 0;
+        }
```

Comment:
newline

Reply:
Above this you mean? Because it needs a check and its 3 friends don't?

Reply:
Oh, I meant between the constructor and properties, helps to keep them distinct